client/checkout/classic/upe-split.js

```diff
@@ -702,11 +702,14 @@ jQuery( function ( $ ) {
 	/**
 	 * Checks if the customer is using a saved payment method.
 	 *
+	 * @param {string} paymentMethodType Stripe payment method type ID.
 	 * @return {boolean} Boolean indicating whether or not a saved payment method is being used.
 	 */
-	function isUsingSavedPaymentMethod() {
+	function isUsingSavedPaymentMethod( paymentMethodType ) {
 		const paymentMethodSelector =
-			'#wc-woocommerce_payments-payment-token-new';
+			'#wc-woocommerce_payments_' +
+			paymentMethodType +
+			'-payment-token-new';
```

Comment:
Just repeating a comment I made on Slack, for the purposes of persistence:

> Just FYI, when Link is enabled AFAIK, the card gateway will appear as a split UPE gateway as well, though its ID will be `woocommerce_payments`, as opposed to `woocommerce_payments_card`. Think you'll need to handle that case inside `isUsingSavedPaymentMethod()` as well.

Reply:
If I understand your suggestion correctly, the main goal is to ensure that `upe-split.js` handles credit card because there are some cases when CC is used as a split UPE gateway. Enabled Link is one of them, but disabled Link & WooPay is another as well. If we are on the same page, then I think we have it now covered. Thanks!